Makefile

```diff
@@ -403,13 +403,14 @@ gen: \
 	provisionersdk/proto/provisioner.pb.go \
 	provisionerd/proto/provisionerd.pb.go \
 	site/src/api/typesGenerated.ts \
-	docs/admin/prometheus.md
+	docs/admin/prometheus.md \
+	coderd/apidoc/swagger.json
 .PHONY: gen
 
 # Mark all generated files as fresh so make thinks they're up-to-date. This is
 # used during releases so we don't run generation scripts.
```

Comment:
sanity check: New API docs (swagger and markdown) are being generated (and committed) during each release? Is this also how we generate our Prometheus docs?

Reply:
Both actions, Prometheus and API docs, use the same target, `make gen`, so every generated doc is stored in the repository. If you see possible drawbacks, we can sync offline.